project/webapp/static/less/thread.less

```diff
@@ -126,18 +128,17 @@ body {
                     }
                 }
             }
-            .add-button {
+            .nav-button {
                 height: 32px;
                 line-height: 32px;
-
                 float: right;
-
-
                 margin-left: auto;
-                .btn{
+                
+                .btn {
                     font-size: 12px;
                     padding: 0 8px;
                     margin-top: 4px;
+                    margin-left: 4px;
```

Comment:
Lets try and use relative units for layout styling. E.g. `em` or `rem` values rather than `px`.